### PR TITLE
fix: Fixed team image not showing on event-types page

### PR DESCRIPTION
**Pull request by pateldivyesh1323**
## What does this PR do?

Fixed team image not showing up on event-types page
- Before:
team-image-not-showing

- After:
team-image-showing

## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

## How should this be tested?
Go on event-types page and check the image of teams.

## Checklist
- I haven't checked if my changes generate no new warnings

### CODE REVIEW

**Review by joeauyeung — DISMISSED** [review]
Thanks for your contribution @pateldivyesh1323. I think there's a way we can take this PR to the next level. In the `EventTypeListHeading` the profile that is being passed contains the proproty `image`.  I think we can use that property and pass it to the `Avatar` component.  This data comes from `packages/trpc/server/routers/viewer/eventTypes/getByViewer.handler.ts`.

The only bug I am seeing is that organization team avatar's are not being rendered now because they don't contain `/team` in the image URL.

**Comment by pateldivyesh1323** [comment]
> Thanks for your contribution @pateldivyesh1323. I think there's a way we can take this PR to the next level. In the `EventTypeListHeading` the profile that is being passed contains the proproty `image`. I think we can use that property and pass it to the `Avatar` component. This data comes from `packages/trpc/server/routers/viewer/eventTypes/getByViewer.handler.ts`.
> 
> The only bug I am seeing is that organization team avatar's are not being rendered now because they don't contain `/team` in the image URL.

Updated as per suggested.

File: apps/web/pages/event-types/index.tsx [hunk 1]
```diff
@@ -720,10 +720,7 @@ const EventTypeListHeading = ({
       <Avatar
         alt={profile?.name || ""}
         href={teamId ? `/settings/teams/${teamId}/profile` : "/settings/my-account/profile"}
-        imageSrc={
-          `${orgBranding?.fullDomain ?? WEBAPP_URL}${teamId ? "/team" : ""}/${profile.slug}/avatar.png` ||
-          undefined
-        }
+        imageSrc={profile?.image || undefined}
```
Comment (Udit-takkar) [inline apps/web/pages/event-types/index.tsx:723]: this isn't working in case of orgs

Comment (Udit-takkar) [reply apps/web/pages/event-types/index.tsx:723]: you can create org using http://localhost:3000/settings/organizations/new

Comment (pateldivyesh1323) [reply apps/web/pages/event-types/index.tsx:723]: It is asking to verify email. I dont know how to setup an org

**Review by Udit-takkar — APPROVED** [review]
LGTM. I fixed the organization avatar. @pateldivyesh1323 Thanks for looking into this

Screenshot 2023-09-09 at 1 55 34 AM

**Pull request merged**